### PR TITLE
Dịch "softmax-regression" - Phần 11

### LINKED ISSUE #725: Dịch "softmax-regression" - Phần 11
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/softmax-regression_vn.md#L540-L597

### PR DESCRIPTION
**Pull request by di-mi-ta**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #725

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/softmax-regression_vn.md [hunk 1]
```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
...
+## Tổng kết
 
 <!--
 * We introduced the softmax operation which takes a vector maps it into probabilities.
 * Softmax regression applies to classification problems. It uses the probability distribution of the output category in the softmax operation.
 * cross-entropy is a good measure of the difference between two probability distributions. It measures the number of bits needed to encode the data given our model.
 -->
 
-*dịch đoạn phía trên*
+* Chúng ta đã giới thiệu về toán tử softmax giúp ánh xạ một vector đầu vào sang các giá trị tạo thành 1 phân phối xác suất.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
* Chúng tôi đã giới thiệu về hàm softmax giúp ánh xạ một vector đầu vào sang các giá trị xác suất. 
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 2]
```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
...
 
 <!--
 * We introduced the softmax operation which takes a vector maps it into probabilities.
 * Softmax regression applies to classification problems. It uses the probability distribution of the output category in the softmax operation.
 * cross-entropy is a good measure of the difference between two probability distributions. It measures the number of bits needed to encode the data given our model.
 -->
 
-*dịch đoạn phía trên*
+* Chúng ta đã giới thiệu về toán tử softmax giúp ánh xạ một vector đầu vào sang các giá trị tạo thành 1 phân phối xác suất. 
+* Hồi quy softmax được áp dụng cho các bài toán phân loại. Nó sử dụng phân phối xác suất của các lớp đầu ra thông qua phép toán softmax.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
* Hồi quy softmax được áp dụng cho các bài toán phân loại. Nó sử dụng phân phối xác suất của các lớp đầu ra thông qua hàm softmax. 
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 3]
```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
...
 <!--
 * We introduced the softmax operation which takes a vector maps it into probabilities.
 * Softmax regression applies to classification problems. It uses the probability distribution of the output category in the softmax operation.
 * cross-entropy is a good measure of the difference between two probability distributions. It measures the number of bits needed to encode the data given our model.
 -->
 
-*dịch đoạn phía trên*
+* Chúng ta đã giới thiệu về toán tử softmax giúp ánh xạ một vector đầu vào sang các giá trị tạo thành 1 phân phối xác suất. 
+* Hồi quy softmax được áp dụng cho các bài toán phân loại. Nó sử dụng phân phối xác suất của các lớp đầu ra thông qua phép toán softmax. 
+* cross-entropy là một phép đánh giá tốt cho sự khác nhau giữa 2 phân phối xác suất. Nó đo lường số lượng bit cần để biểu diễn dữ liệu cho mô hình.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
* Entropy chéo là một phép đánh giá tốt cho sự khác nhau giữa 2 phân phối xác suất. Nó đo lường số lượng bit cần để biểu diễn dữ liệu cho mô hình.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 4]
```diff
@@ -592,8 +598,21 @@ This is equal to the ratio between the number of correct predictions a nd the to
     * Extend this to more than two numbers.
 -->
 
-*dịch đoạn phía trên*
 
+1. Chứng minh rằng độ phân kì Kullback-Leibler $D(p\|q)$ không âm với mọi phân phối $p$ và $q$. Gợi ý: sử dụng bất đẳng thức Jensen hay nghĩa là sử dụng bổ đề $-\log x$ là một hàm lồi.
+2. Chứng minh rằng $\log \sum_j \exp(o_j)$ là một hàm lồi với $o$.
+3. Chúng ta có thể tìm hiểu sâu hơn về sự liên kết giữa các họ hàm mũ và softmax.
+   * Tính đạo hàm cấp hai của hàm mất mát cross-entropy $l(y,\hat{y})$ cho softmax.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
   * Tính đạo hàm cấp hai của hàm mất mát entropy chéo $l(y,\hat{y})$ cho softmax. 
```

**Comment by di-mi-ta** [comment]
Hi @rootonchair, cảm ơn bạn đã review, mình đã xem lại và update lại dựa theo suggest từ bạn!

File: chapter_linear-networks/softmax-regression_vn.md [hunk 5]
```diff
@@ -543,7 +543,7 @@ and (ii) as minimizing our surprise (and thus the number of bits) required to co
 ## Model Prediction and Evaluation
 -->
 
-## *dịch tiêu đề phía trên*
+## Dự đoán và đánh giá mô hình
```
Comment (tanduong) [inline chapter_linear-networks/softmax-regression_vn.md]: Dự đoán mô hình nghe có vẻ mò kim đáy bể. Có thể dịch là 

```suggestion
## Sử dụng và đánh giá mô hình 
```

Hoặc cách nào khác tốt hơn.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 6]
```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
 This is equal to the ratio between the number of correct predictions a nd the total number of predictions.
 -->
 
-*dịch đoạn phía trên*
+Sau khi huấn luyện mô hình hồi quy softmax, với các đặc trưng làm đầu vào bất kì, chúng ta có thể dự đoán xác suất đầu ra ứng với mỗi lớp.
```
Comment (tanduong) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Sau khi huấn luyện mô hình hồi quy softmax, với các đặc trưng đầu vào bất kì, chúng ta có thể dự đoán xác suất đầu ra ứng với mỗi lớp. 
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 7]
```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
 This is equal to the ratio between the number of correct predictions a nd the total number of predictions.
 -->
 
-*dịch đoạn phía trên*
+Sau khi huấn luyện mô hình hồi quy softmax, với các đặc trưng làm đầu vào bất kì, chúng ta có thể dự đoán xác suất đầu ra ứng với mỗi lớp. 
+Thông thường, chúng ta sử dụng lớp với xác suất dự đoán cao nhất làm lớp đầu ra. 
+Một dự đoán được xem là chính xác nếu nó trùng khớp hay tương thích với lớp thật sự (nhãn).
+Ở phần tiếp theo của thí nghiệm, chúng ta sẽ sử dụng độ chính xác để đánh giá chất lượng của mô hình.
```
Comment (tanduong) [inline chapter_linear-networks/softmax-regression_vn.md]: Mình có comment ở thread khác là nên cân nhắc cách dịch chữ accuracy để phân biệt với chữ precision.

File: chapter_linear-networks/softmax-regression_vn.md (hunk 5 above)
Comment (tiepvupsu) [reply chapter_linear-networks/softmax-regression_vn.md]: `Dự đoán mô hình` thì hơi sai nghĩa. Dài hơn một chút nhưng sát hơn 
```suggestion
##  Sử dụng mô hình để dự đoán và đánh giá
```

File: chapter_linear-networks/softmax-regression_vn.md (hunk 7 above)
Comment (tiepvupsu) [reply chapter_linear-networks/softmax-regression_vn.md]: Từ `accuracy` vẫn được dịch là `độ chính xác` trong nhiều tài liệu rồi. Còn từ `precision` trong `precision and recall` thì trong cuốn Machine Learning Yearning bọn mình giữ nguyên.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 8]
```diff
@@ -590,7 +590,7 @@ and (ii) as minimizing our surprise (and thus the number of bits) required to co
 ## Model Prediction and Evaluation
 -->
 
-## *dịch tiêu đề phía trên*
+## Sử dụng mô hình để dự đoán và đánh giá
```
Comment (duythanhvn) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
## Sử dụng Mô hình để Dự đoán và Đánh giá
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 9]
```diff
@@ -600,27 +600,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
...
+Một dự đoán được xem là chính xác nếu nó trùng khớp hay tương thích với lớp thật sự (nhãn).
+Ở phần tiếp theo của thí nghiệm, chúng ta sẽ sử dụng độ chính xác để đánh giá chất lượng của mô hình. 
+Giá trị này là tỉ lệ giữa số mẫu được dự đoán chính xác so với tổng số mẫu được dự đoán.
 
 <!--
 ## Summary
 -->
 
-## *dịch tiêu đề phía trên*
+## Tổng kết
```
Comment (duythanhvn) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
## Tóm tắt
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 10]
```diff
@@ -639,8 +645,21 @@ This is equal to the ratio between the number of correct predictions a nd the to
     * Extend this to more than two numbers.
 -->
 
-*dịch đoạn phía trên*
 
+1. Chứng minh rằng độ phân kì Kullback-Leibler $D(p\|q)$ không âm với mọi phân phối $p$ và $q$. Gợi ý: sử dụng bất đẳng thức Jensen hay nghĩa là sử dụng bổ đề $-\log x$ là một hàm lồi.
+2. Chứng minh rằng $\log \sum_j \exp(o_j)$ là một hàm lồi với $o$.
+3. Chúng ta có thể tìm hiểu sâu hơn về sự liên kết giữa các họ hàm mũ và softmax.
+   * Tính đạo hàm cấp hai của hàm mất mát entropy chéo $l(y,\hat{y})$ cho softmax. 
+   * Tính phương sai của phân phối được cho bởi $\mathrm{softmax}(o)$ và chứng minh rằng nó khớp với đạo hàm cấp 2 được tính ở trên.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
   * Tính phương sai của phân phối được cho bởi $\mathrm{softmax}(o)$ và chứng minh rằng nó khớp với đạo hàm cấp hai được tính ở trên.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 11]
```diff
@@ -700,4 +719,5 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 11 -->
-*
+* Đinh Minh Tân
+* Pham Hong Vinh
```
Comment (duythanhvn) [inline chapter_linear-networks/softmax-regression_vn.md]: Tụi mình đã làm một bảng danh sách các contributors đóng góp cho dự án và để trong tệp dịch, cũng đã đính kèm nó ở trên phần checklist của PR. Nên trước khi điền tên reviewer thì bạn cũng tham khảo qua xem tên reviewer đã có trong đó không để sử dụng cho phù hợp bạn nha.
```suggestion
* Phạm Hồng Vinh
```
